layers/synchronization2.cpp

```diff
@@ -50,19 +50,19 @@ static const VkExtensionProperties kDeviceExtension = {VK_KHR_SYNCHRONIZATION_2_
 
 static const char* const kEnvarForceEnable =
 #if defined(__ANDROID__)
-    "debug.vulkan.sync2.force_enable";
+    "debug.vulkan.synchronization2.force_enable";
 #else
-    "VK_SYNC2_FORCE_ENABLE";
+    "VK_SYNCHRONIZATION2_FORCE_ENABLE";
```

Comment:
I think with global name changes like these, it may break some existing applications using it.

Reply:
This would only affect Vulkan application developers, not Vulkan application users. We can add a notification with next SDK release.

Reply:
Could we detect if the users sets `VK_SYNC2_FORCE_ENABLE` and give a warning? Doing nothing will likely cause an issue for someone.

Reply:
That sounds fair enough, I am adding a log message.